### PR TITLE
Ghouls and muties

### PR DESCRIPTION
**Pull request by AsterixCodix**
## About The Pull Request

Adds in Playable Ghouls, Super Mutants, and Nightkin, along with a small selection of clothing for the oversized Muties. More clothing to come later. Currently, clothes will be issued via the Special tab in the Custom Loadout menus. Eventually, they'll just be issued via a check on the job datums, however... They've caused me enough headaches for now

There are so many new clothing items for them, that listing them would take up an entire page on it's own.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
add: Added playable Nightkin
add: Added playable Super Mutants
add: Added playable Ghouls

add: Mutie clothing (too many to list individually)
/:cl:

### CODE REVIEW

**Comment by AllCorruptingWyrm** [comment]
I love you.

**Comment by AsterixCodix** [comment]
> I love you.

It's not finished yet :p

File: code/modules/mob/living/carbon/human/species_types/ghoul.dm [hunk 1]
```diff
@@ -105,18 +104,18 @@
 		is_healing = FALSE
 		return
 	switch(H.radiation)
-		if(0)
+		if(50)
```
[the remark this answers is not in the document]
Comment (AsterixCodix) [reply code/modules/mob/living/carbon/human/species_types/ghoul.dm]: All good now!

[the remark this answers is not in the document]
Comment (AsterixCodix) [reply code/modules/mob/living/carbon/human/species_types/ghoul.dm]: Should be all good now!

**Pull request merged**